packages/deployment/test/test_kes.js

```diff
@@ -217,7 +218,8 @@ test.serial('getRetainedLambdaAliasNames returns filtered aliasNames '
             + 'on previous version redeployment', async (t) => {
   const kes = t.context.kes;
 
-  kes.config.workflowLambdas = aliasFixture.workflowLambdas;
+  kes.config.workflowLambdas = clonedeep(aliasFixture.workflowLambdas);
+  kes.config.maxNumberOfRetainedLambdas = 2;
```

Comment:
This was accidently passing due to previous test setup.    Should be set for both getRetainedLambdaAliasMetadata tests.